style.css

```diff
@@ -1,211 +1,210 @@
 @import url(http://fonts.googleapis.com/css?family=Jacques+Francois+Shadow);
 @import url(http://fonts.googleapis.com/css?family=Joti+One);
-body
-{
-  background: #000;
-  color: white;
-}
-#stbar
-{
-  border: 1px solid greenyellow;
+body {
+  background: #000
+}
+
+#stbar {
+  border: 1px solid #adff2f;
   width: 100%;
   height: 35px;
   line-height: 30px;
   background: #219;
   position: fixed;
   left: 0;
-  top: 0px;
-  padding: 2px 0px;
+  top: 0;
+  padding: 2px 0;
   font-size: 11px;
-  font-weight: bold;
-  color: #FFFFFF;
+  font-weight: 700;
   text-align: center;
-  -moz-transition: all 0.7s ease-in-out;
-  -webkit-transition: all 0.7s ease-in-out;
-  -o-transition: all 0.7s ease-in-out;
-  -ms-transition: all 0.7s ease-in-out;
-  transition: all 0.7s ease-in-out;
-}
-#stbar:hover
-{  
+  -moz-transition: all .7s ease-in-out;
+  -webkit-transition: all .7s ease-in-out;
+  -o-transition: all .7s ease-in-out;
+  -ms-transition: all .7s ease-in-out;
+  transition: all .7s ease-in-out
```

Comment:
We can remove the prefixes as per [Can I use](https://caniuse.com/#feat=css-transitions).
Please remove the lines other than last one.